### PR TITLE
bumping to 0.13.1

### PR DESCRIPTION
**Pull request by carlodri**
Since I needed `0.13.1` in my workflow, I made this PR. 

Please be kind if something is not right, this is my first contribution attempt on  conda-forge... 😇

### CODE REVIEW

**Comment by Korijn** [comment]
Awesome! If CI passes, everything seems good to me and we'll merge! Thanks a lot.

**Comment by carlodri** [comment]
ok, so looks like:
* I forgot to zero the build number...
* tests for PY27 failed with a (seemingly nasty) segafult [here](https://travis-ci.org/conda-forge/scikit-image-feedstock/jobs/312909692#L9225):

`/Users/travis/miniconda3/conda-bld/scikit-image_1512721956267/test_tmp/conda_test_runner.sh: line 2:  4622 Segmentation fault: 11  "/Users/travis/miniconda3/conda-bld/scikit-image_1512721956267/_t_env/bin/python" -s "/Users/travis/miniconda3/conda-bld/scikit-image_1512721956267/test_tmp/run_test.py"`

Any ideas of what's going wrong?

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
   name: scikit-image
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/s/scikit-image/scikit-image-{{ version }}.tar.gz
-  sha256: 77a636bdc08c7668a15951894548c527f0c8c5c2abc86cb850de17551af51e3e
+  sha256: 353879ddf7483f4621872c49cd9bc8a0ad1c3154ac0670b70799922f4cb899e8
 
 build:
   number: 3
```
Comment (jakirkham) [inline recipe/meta.yaml]: Just so we don't forget, reminder to set this to `0`.

**Comment by jakirkham** [comment]
Since it clears the tests, maybe the `sys.exit` line in the tests is relevant. Unfortunately don't have enough info to say why. Don't think it is any of the `conda inspect` lines, but maybe. Perhaps you could try getting `python` to run `run_test.py` with verbose mode to get a better idea of what is happening. This probably requires renaming it so `conda-build` doesn't pick it up by default and running it in the `commands` of `meta.yaml`.

**Comment by carlodri** [comment]
I'm kind of stuck here. It seems clear that the culprit is indeed the `sys.exit` line, but googling possible segfaults there only brought me [here](https://bugs.python.org/issue1856), which is a tough beast for me...

Isn't this the case where one should use `gdb` to get more info on the segfault?

**Comment by carlodri** [comment]
I would be very happy to know _why_ allowing `networkx` 2.0 solved the segfault on `PY27`.... 😮 

anyways, looks like a green light!

**Comment by jakirkham** [comment]
Hmm...sounds suspicious. NetworkX is pure Python AFAIK. Perhaps the segfault is a sporadic issue?

**Comment by carlodri** [comment]
could well be: correct me if I'm wrong, but as far as I understand, some issue with `sys.exit` related to existing deamon threads upon exit were never completely solved in the python 2.7 branch because fixing them would have caused regressions in other python components. Anyway, problem solved, at least for now...

**Comment by carlodri** [comment]
@Korijn can this PR be merged please?

**Comment by carlodri** [comment]
@Korijn thanks!

**Pull request merged**